site/_docs/reference.md

```diff
@@ -2728,6 +2729,7 @@ BigQuery's type system uses confusingly different names for types and functions:
 | b | FORMAT_DATETIME(string, timestamp)             | Formats *timestamp* according to the specified format *string*
 | b | FORMAT_TIME(string, time)                      | Formats *time* according to the specified format *string*
 | b | FORMAT_TIMESTAMP(string timestamp)             | Formats *timestamp* according to the specified format *string*
+| s | GETBIT(value, position)                        | Returns the bit (0 or 1) value at the specified *position* of *value*. The positions are numbered from right to left, starting at zero. The *position* argument cannot be negative
```

Comment:
Could be simplify to 'Equivalent to `BIT_GET(value, position)`' ?

Reply:
It would be more concise. I will fix it soon.